cmd/image-builder-db-test/main_test.go

```diff
@@ -435,7 +435,10 @@ func testBlueprints(t *testing.T) {
 	require.Equal(t, entries[1].Version, updated.Version)
 
 	err = d.InsertBlueprint(uuid.New(), uuid.New(), ORGID1, ANR1, "unique name", "unique desc", body)
-	entries, count, err := d.FindBlueprints(ORGID1, "unique", 100, 0)
+	entries, count, err := d.FindBlueprints(ORGID1, "", 100, 0)
+	require.NoError(t, err)
+	require.Equal(t, 1, count)
```

Comment:
We've got 2 created prior this test already, I'd just bump it to 3 here, but we might want to think about splitting this test up in some refactoring PR IMO :)